ui/app/templates/components/generated-item-list.hbs

```diff
@@ -61,14 +59,10 @@
         this.itemType
       }} will be listed here. Create your first {{this.itemType}} to get started."
     >
-      {{! template-lint-configure no-curly-component-invocation "warn" }}
-      {{! template-lint-configure no-link-to-positional-params "warn" }}
-      {{#link-to "vault.cluster.access.method.item.create" this.itemType class="link"}}
+      <LinkTo @route="vault.cluster.access.method.item.create" @model={{this.itemType}} class="link">
```

Comment:
does the class="link" here do any styling still. e.g. is it still needed?

Reply:
There is a `link` class in the `generic.scss` which appears to change the appearance from our standard link styling.